### PR TITLE
Story Editor: Design Updates to Panel

### LINKED ISSUE #4851: Workspace: Design Updates to Text Panel
## Feature Description

In the screenshot, design mock on the left and what's currently in the editor on the right. Can we make these 2 adjustments:
Decrease the left and right gutter padding is 16px for all the element panel. I'm not sure how this will impact the scrollbar (ie if scrollbar is additional px space?).
The space between the text set boxes are 12px.

Additionally drop the "Add New Text" blue link

## Additional Context

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance Criteria

The text panel should match the mocks on the left.

### PR DESCRIPTION
**Pull request by maxyinger**
## Summary
This PR implements the style updates in this ticket so the panes in the editor are closer to the designs. 

Also updated margin on panel captions & updated preset font weights to match figma designs here:
https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories---Post-Stable?node-id=2%3A31374

## Relevant Technical Choices
Just style updates

## To-do
I can take care of updating the accent color to purple here if it's not already documented in another ticket

## User-facing changes
Updated spacing in the editor panels

## Testing Instructions
Open up the editor and see that the design panels spacing between it's contents reflects the designs in figma better

---

Fixes #4851

### CODE REVIEW

File: assets/src/edit-story/components/library/panes/text/textPane.js [hunk 1]
```diff
@@ -64,14 +57,7 @@ function TextPane(props) {
         />
       )}
 
-      <Section
-        title={__('Presets', 'web-stories')}
-        titleTools={
```
Comment (swissspidy) [inline assets/src/edit-story/components/library/panes/text/textPane.js:69]: This is the only place where `titleTools` appears to have been used.

Let's remove support for this prop from the `Section` component too

**Review by BrittanyIRL — APPROVED** [review]
LGTM 🚀 

Confirmed that:
- on the text panel the left and right gutter padding is 16px
- there is a vertical space of 12px between text set boxes
- "Add New Text" button is gone

File: assets/src/edit-story/components/library/text/fontPreview.js [hunk 2]
```diff
@@ -68,7 +68,7 @@ function FontPreview({ title, element, onClick }) {
   }, [font, fontWeight, content, maybeEnqueueFontStyle]);
 
   return (
-    <Preview onClick={onClick}>
+    <Preview onClick={onClick} fontSize={fontSize}>
```
Comment (barklund) [inline assets/src/edit-story/components/library/text/fontPreview.js]: Why is this here - the `fontSize` prop?

File: assets/src/edit-story/components/library/panes/text/textPresets.js [hunk 3]
```diff
@@ -67,7 +67,7 @@ const PRESETS = [
         'Heading 2',
         'web-stories'
       )}</span>`,
-      fontWeight: FONT_WEIGHT.BOLD,
+      fontWeight: FONT_WEIGHT.MEDIUM,
```
Comment (barklund) [inline assets/src/edit-story/components/library/panes/text/textPresets.js:70]: This property is actually completely ignored now (and has been for quite a while). The font-weight has to be supplied inline in the `content` HTML above - and that's still set to 700. Please remove this property from all the presets and fix the inline property in the HTML instead, thanks!

File: assets/src/edit-story/components/library/common/section.js [hunk 4]
```diff
@@ -30,55 +30,27 @@ const Container = styled.div`
...
+export default function Section({ title, children, ...rest }) {
   return (
     <Container {...rest}>
-      {(title || titleTools) && (
-        <TitleAndTools>
-          {title && <Title>{title}</Title>}
-          {titleTools && <Tools>{titleTools}</Tools>}
-        </TitleAndTools>
-      )}
+      {title && <Title>{title}</Title>}
```
Comment (barklund) [inline assets/src/edit-story/components/library/common/section.js]: `title` is required, so no need to test for it.

**Review by barklund — APPROVED** [review]
This looks good and is reducing code - I always appreciate that! 🎉 

However, this is a pretty big change to the layout of the library panes, so take extra care when reviewing [the 69 changes in Percy](https://percy.io/google/web-stories-wp/builds/7216759)!

File: assets/src/edit-story/components/library/text/fontPreview.js (hunk 2 above)
Comment (maxyinger) [reply assets/src/edit-story/components/library/text/fontPreview.js]: Oh good catch. I was playing with something but forgot to remove. will take it out

File: assets/src/edit-story/components/library/test/text/textPane.js [hunk 5]
```diff
@@ -1,92 +0,0 @@
...
-
-jest.mock('../../useLibrary');
-jest.mock('../../../../app/font/useFont');
-import useFont from '../../../../app/font/useFont';
-import useLibrary from '../../useLibrary';
-import FontContext from '../../../../app/font/context';
-import fontsListResponse from '../../../fontPicker/test/fontsResponse.json';
-import { curatedFontNames } from '../../../../app/font/curatedFonts';
-
-describe('TextPane', () => {
```
Comment (barklund) [inline assets/src/edit-story/components/library/test/text/textPane.js]: Wait, why are these tests being removed? Could they not simply be amended?

Comment (maxyinger) [reply assets/src/edit-story/components/library/test/text/textPane.js]: I removed it cus it's just the one test for the `Add new text` button that was removed.

But yea, looking at it again I can just perform the same test but on clicking a preset instead of the `Add new text button. will do that unless you think otherwise

File: assets/src/edit-story/components/library/panes/text/textPresets.js (hunk 3 above)
Comment (maxyinger) [reply assets/src/edit-story/components/library/panes/text/textPresets.js:70]: Looks like this isn't ignored in the actual preview button content, but is ignored for rendering the element.

**Review by carloskelly13 — APPROVED** [review]

**Review by embarks — APPROVED** [review]
